src/resources/common.yml

```diff
@@ -489,6 +486,15 @@ common:
   filter:
     en: Filter
     de: Filter
+  playerSettings:
+    en: Player settings
+    de: Player-Einstellungen
+  initialVolume:
+    en: Initial volume
+    de: Anfangslautstärke
+  multitrackInitialVolumeInfo:
+    en: This is the initial volume of the player, applied over all tracks. Available only when there is more than one track. [but better]
```

Comment:
This is the initial mix volume

Dies ist die initiale Mix-Lautstärke

Reply:
Changed!